src/script/script.cpp

```diff
@@ -276,18 +277,16 @@ bool CScript::HasValidOps() const
     return true;
 }
 
-bool GetScriptOp(CScriptBase::const_iterator& pc, CScriptBase::const_iterator end, opcodetype& opcodeRet, std::vector<unsigned char>* pvchRet)
+bool GetScriptOp(Span<const unsigned char>& script, opcodetype& opcodeRet, std::vector<unsigned char>* pvchRet)
```

Comment:
AFAICT span end can change whereas before the end iterator couldn't change. Is this worth noting?

Reply:
Noted.

Reply:
👀

Reply:
I'm not sure what you're asking here. Yes, it's a change, but I don't think it significant enough to point out in the commit, and certainly not in the code itself.

If you mean that it should be pointed out to other reviewers, I agree - and you just did :)

Reply:
Ok then.